doc/template-user-guide.md

```diff
@@ -54,19 +54,19 @@ oc new-app --file pkg/3scale/amp/auto-generated-templates/amp/amp.yml \
 
 | Parameter Name | Description | Default |
 | :--- | :---| :--- |
-| **AMP_RELEASE** | AMP release tag | 2.7 |
+| **AMP_RELEASE** | AMP release tag | 2.9 |
 | **APP_LABEL** | Used for object app labels | 3scale-api-management |
 | **TENANT_NAME** | Default tenant prefix name. *-admin* suffix will be appended | 3scale |
 | **RWX_STORAGE_CLASS** | The Storage Class to be used by ReadWriteMany PVCs | 'null' |
-| **AMP_BACKEND_IMAGE** | 3scale Backend component docker image URL | quay.io/3scale/apisonator:nightly |
-| **AMP_ZYNC_IMAGE** | 3scale Zync component docker image URL | quay.io/3scale/zync:nightly |
-| **AMP_APICAST_IMAGE** | 3scale Apicast component docker image URL | quay.io/3scale/apicast:nightly |
-| **AMP_SYSTEM_IMAGE** | 3scale System component docker image URL | quay.io/3scale/porta:nightly |
-| **ZYNC_DATABASE_IMAGE** | Zync's PostgreSQL image to use | centos/postgresql-10-centos7 |
-| **MEMCACHED_IMAGE** | Memcached image to use | memcached:1.5 |
+| **AMP_BACKEND_IMAGE** | 3scale Backend component docker image URL | registry.redhat.io/3scale-amp2/backend-rhel7:3scale2.9 |
+| **AMP_ZYNC_IMAGE** | 3scale Zync component docker image URL | registry.redhat.io/3scale-amp2/zync-rhel7:3scale2.9 |
+| **AMP_APICAST_IMAGE** | 3scale Apicast component docker image URL | registry.redhat.io/3scale-amp2/apicast-gateway-rhel7:3scale2.9 |
```

Comment:
APIcast gateway is rhel8 based and not rhel7 based now

Reply:
done